quinn-proto/src/connection/streams/state.rs

```diff
@@ -434,20 +432,14 @@ impl StreamsState {
                 .get_or_create()
                 .reset_stream
                 .push((id, error_code));
-            frame::ResetStream {
+            let frame = frame::ResetStream {
                 id,
                 error_code,
                 final_offset: VarInt::try_from(stream.offset()).expect("impossibly large offset"),
-            }
-            .encode(buf);
+            };
+            frame.encode(buf);
             #[cfg(feature = "qlog")]
-            qlog.frame(QuicFrame::ResetStream {
-                stream_id: id.into(),
-                error_code: error_code.into(),
-                final_size: stream.offset(),
-                length: None,
-                payload_length: None,
-            });
+            qlog.frame(&Frame::ResetStream(frame));
```

Comment:
The function has an inner `cfg` now, I think you can also remove the cfg here and the compiler should figure out this is all a noop and remove the call?

Also in the other locations in this file.